pinc/wordcheck_engine.inc

```diff
@@ -311,21 +312,11 @@ class BadWordAccumulator
  */
 function get_bad_words_via_external_checker($input_words_w_freq, $languages)
 {
-    global $aspell_temp_dir;
     global $aspell_executable, $aspell_prefix;
 
-    $messages = [];
-
-    $tmp_file_path = tempnam($aspell_temp_dir, "pagetext-");
-
     $tmp_file_text = join("\n", array_keys($input_words_w_freq));
-    $wasWritten = file_put_contents($tmp_file_path, $tmp_file_text);
-    unset($tmp_file_text);
-    if ($wasWritten === false) {
-        $messages[] = sprintf(_("Error: unable to write text to temp file: %s"), $tmp_file_path);
```

Comment:
Do we need to remove the orphaned translations for this string? Or is that going to be handled when we next update the translation files?

Reply:
If the string isn't in the code, the translation doesn't get displayed, because it doesn't map to anything that gets displayed.

Even once the POT is regenerated, the orphaned translation remains in the POT until the old strings are purged, but, again, there's nothing they map to, unless one of them gets re-added to the code.

Thanks for the reminder -- it's past time for me to regenerate the POT files, etc.